node/mock/factory/consensusComponentsMock.go

```diff
@@ -0,0 +1,65 @@
+package factory
```

Comment:
1. When adding mocks/stubs, never add them in the same package/level with the productive file. We have `testscommon` package for them.
2. This should be deleted, is already duplicated with existing mocks, like **`ConsensusCoreMock`**. Please use existing mocks